nucypher/cli/options.py

```diff
@@ -46,7 +46,9 @@
 option_event_name = click.option('--event-name', help="Specify an event by name", type=click.STRING)
 option_federated_only = click.option('--federated-only/--decentralized', '-F', help="Connect only to federated nodes", is_flag=True, default=None)
 option_force = click.option('--force', help="Don't ask for confirmation", is_flag=True)
-option_gas_price = click.option('--gas-price', help="Use this gas price (in GWEI)", type=GWEI)
+option_gas_price = click.option('--gas-price', help="Set a static gas price (in GWEI)", type=GWEI)
+option_gas_strategy = click.option('--gas-strategy', help="Operate with a specified gas price strategy", type=click.STRING)  # TODO: GAS_STRATEGY_CHOICES
+option_max_gas_price = click.option('--max-gas-price', help="Maximum acceptable gas price in Gwei", type=GWEI)
```

Comment:
Matching format of `option_gas_price`

```suggestion
option_max_gas_price = click.option('--max-gas-price', help="Maximum acceptable gas price (in GWEI)", type=GWEI)
```